src/components/wizard/wizard.component.ts

```diff
@@ -193,6 +196,18 @@ export class WizardComponent implements AfterViewInit, OnDestroy {
         }
     }
 
+     /**
+     * If the validation is false then disable the next button
+     * and do not allow progression onto the next step however
+     * if the validation is set to true then the next button
+     * disable the next button and give a greyed out
+     * appearance as an indicator
+     */
+    isNextDisabled(): boolean {
+        const { valid, disableNextWhenInvalid } = this.getCurrentStep() as WizardStepComponent;
```

Comment:
You don't need to cast the return value, the return type of `getCurrentStep` is `WizardStepComponent`.